cli/js/lib.deno.ns.d.ts

```diff
@@ -1892,8 +1892,18 @@ declare namespace Deno {
    * Requires `allow-run` permission. */
   export function run<T extends RunOptions = RunOptions>(opt: T): Process<T>;
 
-  interface InspectOptions {
+  export interface InspectOptions {
+    /** Traversal depth for nested objects. Defaults to 4. */
     depth?: number;
+    /** Sort Object, Set and Map entries by key. Defaults to false. */
+    sorted?: boolean;
+    /** Add a trailing comma for multiline collections. Defaults to false. */
+    trailingComma?: boolean;
+    /** Try to fit more than one entry of a collection on the same line.
+     * Defaults to true. */
+    compact?: boolean;
+    /** The maximum number of iterable entries to print. Defaults to 100. */
+    iterableLimit?: number;
```

Comment:
I'm not particularly happy to add a bunch of options that must be maintained forever. Is it at all possible to exclude these and still achieve your goal of fixing up the assert diff output?

Reply:
> Is it at all possible to exclude these and still achieve your goal of fixing up the assert diff output?

Not really, I only added those necessary. Note that Node has all of these options except `trailingComma`, which they likely would in the future.

Reply:
@ry do you think it would be better if we combine all these options into a single option, say “pretty: boolean”?

Reply:
To be honest, I rather all these options provided here become the default, the “console.log” in Node.js frustrate me that it don’t print out everything. I always ended up using “console.log(JSON.stringify(x, null, 2))” instead, which is obviously worse because there’s no colouring.

Reply:
@nayeemrmn what do you think about the suggestion above?

Reply:
If we decide to diverge from Node's defaults, I'd prefer if it wasn't attributed to this PR. Propose as a follow up.

I would strongly dislike always wrapping by default for small objects. Sorting by default would lead to surprises -- the iteration order should be the default since it corresponds to behaviour. I don't particularly mind the others. Note that we have to consider browser inspector defaults, too.

Reply:
If that is the case I suggest that we remove options that can be made default (e.g. trailingComma, infinite depth), and leave the rest as options.

Reply:
> Not really, I only added those necessary. Note that Node has all of these options except trailingComma, which they likely would in the future.

Ok. Fair enough. I think this is ok. Let me give it another look over, but I think we can land it.

Reply:
@ry Are you going to land this in 1.1.4?